### PR TITLE
[DDW-971] Implement new Wallet Receive screen

### PR DESCRIPTION
**Pull request by daniloprates**
This PR implements the new Wallet Receive screen.

## Todos

- [x] Main screen changes
- [x] Share dialog
- [x] Japanese translation
- [x] Update Storybook
- [x] Update tests

### Wallet Receive
E1

### Wallet Receive - PDF dialog
E2

### Address PDF
E3

---

## Testing Checklist

- [Slack QA thread](https://github.com/input-output-hk/daedalus/pull/1700)
- [Copy and translation](https://docs.google.com/document/d/10fLIHS6pHlcob7rnmzoJRkxQUeF_rtfvYmgN-CfnVOo/edit#)
- [ ] Test

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by a-rukin** [comment]
image
@daniloprates styles fix would be in a separate task, right?

**Comment by nikolaglumac** [comment]
@daniloprates on minimum window width the addresses and "copy" icon overlap...

Screenshot 2019-11-28 at 09 14 36

File: source/renderer/app/components/wallet/receive/Address.js [hunk 1]
```diff
@@ -5,43 +5,132 @@ import CopyToClipboard from 'react-copy-to-clipboard';
...
+              <span className={styles.copyAddress}>
+                <SVGInline svg={iconCopy} className={styles.copyIcon} />
+                <span className={styles.copyAddressLabel}>
+                  {copyAddressLabel}
+                </span>
               </span>
-            </span>
-          </CopyToClipboard>
+            </CopyToClipboard>
+          ) : null}
```
Comment (tomothespian) [inline source/renderer/app/components/wallet/receive/Address.js]: @daniloprates you don't need 2 clauses with `null` fallback. You can put all together like:
`{isIncentivizedTestnet ? (...code with copy) : (... code with share)}`

**Comment by a-rukin** [comment]
image
In story book (before some width) it's just cutting the words, when reaching width normal crop is applied

**Pull request merged**